packages/api/src/module/read/course-progress/course-progress.read-module.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { CourseProgress } from '@prisma/client';
+import { AsyncReturnType } from 'type-fest';
+import { v4 as uuid } from 'uuid';
+import waitForExpect from 'wait-for-expect';
+
+import { initTestModule } from '@/common/test-utils';
+import { ApplicationEvent } from '@/module/application-command-events';
+import { LearningMaterialsUrlWasGenerated } from '@/module/events/learning-materials-url-was-generated.domain-event';
+import { TaskWasCompleted } from '@/module/events/task-was-completed.domain-event';
+import { EventStreamName } from '@/write/shared/application/event-stream-name.value-object';
+
+async function courseProgressTestModule() {
+  const { prismaService, close, eventOccurred } = await initTestModule();
+
+  async function addExampleData({
```

Comment:
It is this necessary?

Reply:
I've tried to remove it, and something strange has happend in tests:
```
This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
Error: The operation was canceled.
```

Any ideas?

Reply:
@KonradSzwarc Do you know how to find what is wrong?